packages/app-config-writer/README.md

```diff
@@ -0,0 +1,41 @@
+# @sap-ux/app-config-writer
+
+The `@sap-ux/app-config-writer` adds smartlinks configuration for a SAP UX project. 
+
+## Installation
+Npm
+`npm install --save @sap-ux/app-config-writer`
+
+Yarn
+`yarn add @sap-ux/app-config-writer`
+
+Pnpm
+`pnpm add @sap-ux/app-config-writer`
+
+## Usage
+
```

Comment:
Could you please also give an example in our to use it together with the `@sap-ux/create` module e.g. `npm init @sap-ux app-config add smartlinks` ?

Reply:
Thanks, added code example `npm init @sap-ux add smartlinks-config` [R25-28](https://github.com/SAP/open-ux-tools/pull/913/files#diff-e422e89a638c2b8978be2545eace180c17a2a35a0659610979f9e840b5a1dd50R25-R28)